invenio_app_rdm/config.py

```diff
@@ -702,6 +700,29 @@ def files_rest_permission_factory(obj, action):
 ]
 """Template names for detail view sidebar components"""
 
+FILES_INTEGRITY_REPORT_TEMPLATE = (
```

Comment:
These 3 are actually being picked up. Mainly because they are in a config module entry point (not by the ext.py). 

**Major: Should I prefix them with `APP_RDM` for consistency with the above?**

Reply:
I think yes, we should keep them consistent